### DISCUSSION TITLE
How to show the Lighthouse and CheckBot benchmark results on a wordpress page?

### DISCUSSION THREAD

**Discussion by wadid** [GoogleChrome/lighthouse#13702] [Q&A]
Basically I want to build a wordpress site which shows different seo metrics like e.g. the lighthouse and checkbot metrics. In the frontend. So a customer can access that wordpresssite and see the lighthouse results (to the customers specific url).

I am struggling to find a way how and where to grab the lighthouse data (with the nice graphics) to include it on the site.

Any ideas how to do this? Do I need an API or do I grab it from an html?

**Comment by adamraine** [comment]
> Basically I want to build a wordpress site which shows different seo metrics like e.g. the lighthouse and checkbot metrics. In the frontend. So a customer can access that wordpresssite and see the lighthouse results (to the customers specific url).

You can use this [PSI API](https://developers.google.com/speed/docs/insights/v5/get-started). The response from the API includes a Lighthouse result JSON.

> I am struggling to find a way how and where to grab the lighthouse data (with the nice graphics) to include it on the site.

Our report renderer is bundled in the Lighthouse npm package. [web.dev uses this to render a LH report.](https://github.com/GoogleChrome/web.dev/blob/62c6450285fbfd6b4850af345f4ff6264bddec7a/src/lib/components/LighthouseViewer/index.js#L3)
